src/ResourceManager/ServiceBus/ChangeLog.md

```diff
@@ -20,6 +20,157 @@
+   
+   **Get-AzureRmServiceBusNamespaceKey**, 
+   - 'Get-AzureRmServiceBusNamespaceKey' cmdlet is removed in current build. Please use the New cmdlet 'Get-AzureRmServiceBusKey'
+   
+   **New-AzureRmServiceBusNamespaceAuthorizationRule**,
+   - 'New-AzureRmServiceBusNamespaceAuthorizationRule' cmdlet is removed in current build. Please use the New cmdlet 'New-AzureRmServiceBusAuthorizationRule'
+   
+   **Remove-AzureRmServiceBusNamespaceAuthorizationRule**,
+   - 'Remove-AzureRmServiceBusNamespaceAuthorizationRule' cmdlet is removed in current build. Please use the New cmdlet 'Remove-AzureRmServiceBusAuthorizationRule'
+   
+   **Set-AzureRmServiceBusNamespaceAuthorizationRule**
+   - 'Set-AzureRmServiceBusNamespaceAuthorizationRule' cmdlet is removed in current build. Please use the New cmdlet 'Set-AzureRmServiceBusAuthorizationRule'
+   
+   
+   The following properties are removed in this release:
+   
+   ** NamespceAttributes **
+   - Status
+   - Enabled
+   
+   ```powershell
+    # Old
+	# The $namespace has Status and Enabled property 
+    $namespace = Get-AzureRmServiceBusNamespace <parameters>
+	$namespace.Status
+	$namespace.Enabled
+    # New
+
+    # The call remains the same, but the returned values NameSpace object will not have the ResourceGroupName property    
+    $namespace = Get-AzureRmServiceBusNamespace <parameters>
+    
+    ```
+   
+   
+   ** Queue **
+   - EntityAvailabilityStatus
+   - EnableBatchedOperations
+   - IsAnonymousAccessible
+   - SupportOrdering
+   
+   ```powershell
+   
+    # Old
+	# The $queue has EntityAvailabilityStatus, EnableBatchedOperations, IsAnonymousAccessible and SupportOrdering property 
+    $queue = Get-AzureRmServiceBusQueue <parameters>
+	$queue.EntityAvailabilityStatus
+	$queue.EnableBatchedOperations
+	$queue.IsAnonymousAccessible
+	$queue.SupportOrdering	
+
+    # New
+
+    # The call remains the same, but the returned values Queue object will not have the EntityAvailabilityStatus, EnableBatchedOperations, IsAnonymousAccessible and SupportOrdering properties    
+    $queue = Get-AzureRmServiceBusQueue <parameters>
+    
+    ```
+   
+   ** Topic **
+   - Location
+   - IsExpress
+   - IsAnonymousAccessible
+   - FilteringMessagesBeforePublishing
+   - EnableSubscriptionPartitioning
+   - EntityAvailabilityStatus
+   
+   
+    ```powershell
+   
+    # Old
+	# The $topic has EntityAvailabilityStatus, EnableSubscriptionPartitioning, IsAnonymousAccessible, IsExpress, Location and FilteringMessagesBeforePublishing property 
+    $topic = Get-AzureRmServiceBusTopic <parameters>
+	$topic.EntityAvailabilityStatus
+	$topic.EnableSubscriptionPartitioning
+	$topic.IsAnonymousAccessible
+	$topic.IsExpress
+	$topic.FilteringMessagesBeforePublishing
+	$topic.Location
+
+    # New
+
+    # The call remains the same, but the returned values Topic object will not have the EntityAvailabilityStatus, EnableBatchedOperations, IsAnonymousAccessible and SupportOrdering properties    
```

Comment:
The properties that you say are missing here don't match the ones you say existed in the old